### PR TITLE
Add Boilerplate Bot Code 🤖

### PR DESCRIPTION
**Pull request by samuelsogbesan**
Added the boilerplate bot code from following the guide on the [Discord repo](https://github.com/discordjs/discord.js#example-usage).

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const Discord = require('discord.js');
+require('dotenv').config();
```
Comment (bailscodesstuff) [inline index.js:2]: whats this for

**Review by bailscodesstuff — APPROVED** [review]
i get u

File: index.js (hunk 1 above)
Comment (samuelsogbesan) [reply index.js:2]: basically a .env file is a file used to store secrets (such as passwords or ids) that you don't want to publish publically. 
the contents of the file are literally a bunch of key val pairs like this:
<code>
SECRET_KEY = SECRET_VALUE
</code>

['dotenv'](https://www.npmjs.com/package/dotenv) is a small library that parses the .env file and makes the things in the file accessible as variables.

it actually does this by making them accessible via the global object <code>process.env</code>

e.g. <code>process.env.SECRET_KEY</code>

<code>require()</code> is syntax we use to import modules (other files / libraries basically) in Javascript.

so what that line does is imports dotenv, and calls the <code>config()</code> method, which just does some configuration, including parsing the .env and adding its contents to the global i mentionned earlier.

@bailslearnsstuff

**Comment by samuelsogbesan** [comment]
> 
> 
> i get u

woop 🕺

File: index.js (hunk 1 above)
Comment (bailscodesstuff) [reply index.js:2]: NICE

**Pull request merged**